tests/TestCase/Mailer/Transport/SmtpTransportTest.php

```diff
@@ -855,16 +856,18 @@ public function testSerializeCleanupSocket(): void
         $this->socket->expects($this->once())
             ->method('write')
             ->with("EHLO localhost\r\n");
+        $this->socket->expects($this->once())
+            ->method('isConnected')
+            ->willReturn(true);
 
         $smtpTransport = new SmtpTestTransport();
         $smtpTransport->setSocket($this->socket);
         $smtpTransport->connect();
 
+        $this->assertTrue($smtpTransport->connected());
```

Comment:
The `connect()` call above already calls `connected()`.

Reply:
Ah, silly me

Reply:
I don't think I can test what I want to using the mock anyway. Can't expect SmtpTransport instance created after unserialization to have the same mock instance of the socket.